configs/components/leatherman.json

```diff
@@ -1 +1 @@
-{"url": "git://github.com/puppetlabs/leatherman.git", "ref": "0.8.1"}
```

Comment:
0.9.0 is necessary if we're building a newer version of Facter. If we're sticking to facter#stable, we can probably stay at 0.8.1.